TicTacToe.cpp

```diff
@@ -77,3 +78,36 @@ int TicTacToe::checkVictoryHorizontal(int teamSide)
     }
     return point;
 }
+
+
+void TicTacToe:viewAll()
+{
+	int hold[26];   \\ create array to hold field integers 
```

Comment:
Hrm. Could you explain your rationale for making a copy of the board in array form? I'm not sure why you wouldn't just use `field[r][c][z]` later on to print out the field.

Reply:
Looking back at it, I mainly did so because I wanted a casual way of referencing the board repeatedly and I didn't think that this form had any issue associated with it. I guess the for loops are unnecessarily taking time, and with added consideration, it probably wasn't the smart decision. However, I'm not sure if it's problematic enough to redo the script. Let me know what your thoughts are

Reply:
I see. Well, like you say there is a bit of a performance penalty (and potential bug risk) to extraneously copying days structures around, but I wouldn't say its mandatory to fix here. We're not making a high performance game engine here... Let's leave that to the guys who make Unreal lol.

But technically speaking, I'm not sure it would be that much work to strip out the temporary array and replace the 27 (I think, right?) instances of `hold` with `field`. Ultimately, it's your call. If you'd rather move on, I'll gladly merge this.